cmd/apiserver/app/server/run_server.go

```diff
@@ -183,7 +183,7 @@ func (c checkEtcdConnectable) Check(_ *http.Request) error {
 	serverReachable, err := preflight.EtcdConnection{ServerList: c.ServerList}.CheckEtcdServers()
 
 	if err != nil {
-		glog.Error("etcd checker failed with err: %v", err)
+		glog.Errorf("etcd checker failed with err: %v", err)
```

Comment:
Formatter looks wrong to me too. Agree it should be `Errorf`. Very not sure how it got through the build. I know at some points we have had verification that didn't fail the build.

I don't think the `/healthz` healthchecker endpoint logs anything internally, so I'm raising it to the operator if they ever check the logs that etcd might be down or not connectable.

This code gets called whenever `GET /healthz` happens. Usually this is done during the liveness/readiness probes by k8s infra. So it could be called repeatedly over a long period of time, but as long as it never fails, nothing should be logged. We could maybe change the level of the info at the top to be a lower level.